blank-page/template-parts/content/content-singular.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Template part for displaying posts
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
+ *
+ * @package Blank Page
+ * @since 1.0
+ */
+
+?>
+
+<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+
+	<?php seedlet_post_thumbnail(); ?>
+
+	<div class="entry-content">
+		<?php
+		the_content(
+			sprintf(
+				wp_kses(
+					/* translators: %s: Name of current post. Only visible to screen readers */
+					__( 'Continue reading<span class="screen-reader-text"> "%s"</span>', 'seedlet' ),
```

Comment:
I don't think it matters since we won't be submitting to the theme directory, but should these text domains be `blank-page`? Or are they left as Seedlet in order to reference Seedlet's translation file?

Reply:
Yeah, they should probably be blank page. I'll update.